testing.md

```diff
@@ -7,6 +7,7 @@ Manual tests to execute before plugin releases.
 Pre-reqs: Run through the [flutter setup](https://flutter.io/setup/) and
 [flutter getting started](https://flutter.io/getting-started/) guides.
 
+* Be sure you're on the  `alpha` branch.
```

Comment:
If we align our release schedules, we'll probably want to test against the up-coming alpha (the master branch).